skaffold.yaml.template

```diff
@@ -36,7 +36,7 @@ deploy:
     # datarepo-api
     - name: TEMP-jade-datarepo-api
       chartPath: https://github.com/broadinstitute/datarepo-helm/releases/download/datarepo-api-0.0.23/datarepo-api-0.0.23.tgz
```

Comment:
these need to bump bumped too, I'm not convinced this value is the right one tho since its so specific